amazon-ecr-credential-helper.spec

```diff
@@ -118,6 +136,11 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Fri Jan 14 2022 Austin Vazquez <macedonv@amazon.com> - 0.6.0-1
+- Added support for AWS SSO
+- Added support to assume roles via EC2 instance metadata
+- Added support for Apple Silicon
```

Comment:
@samuelkarp Did you "curate" ChangeLog entries before or was is just as is? I wonder whether we should mention Apple Silicon or not.

Reply:
FWIW, this wording is inline with the release notes [here](https://github.com/awslabs/amazon-ecr-credential-helper/blob/main/CHANGELOG.md#060).

Reply:
> Did you "curate" ChangeLog entries before

Yes, generally the entries in the RPM changelog only cover what's relevant to the RPM.

Reply:
Should be resolved in latest. 👍🏼